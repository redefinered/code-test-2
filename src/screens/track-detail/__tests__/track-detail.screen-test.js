import React from 'react';
import TrackDetailScreen from '../track-detail.screen';

import renderer from 'react-test-renderer';

describe('search-bar tests', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<TrackDetailScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
