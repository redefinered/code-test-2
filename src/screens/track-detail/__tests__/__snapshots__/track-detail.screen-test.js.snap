// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search-bar tests renders correctly 1`] = `
<View
  style={
    Object {
      "padding": 15,
    }
  }
>
  <Image
    source={
      Object {
        "height": 100,
        "uri": undefined,
        "width": 100,
      }
    }
    style={
      Object {
        "marginBottom": 10,
      }
    }
  />
  <Text
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 5,
      }
    }
  />
  <Text>
    By undefined
  </Text>
  <Text>
    Album: undefined
  </Text>
  <Text>
    Release date: Aug 4, 2022
  </Text>
</View>
`;
